src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+/**
+ * Sorts all the data in the address book in alphabetical order of Name
+ */
+public class SortCommand extends Command{
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book in alphabetical order of their name.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "All persons in address book sorted by name";
+    public static final String MESSAGE_FAILURE = "Unable to sort persons in address book.";
```

Comment:
Good job on a whole! Perhaps you could consider changing this line to something like "There are no persons in address book to sort." to be more specific and more user-friendly?